atlp-rwanda/e-comm-team-techsmith-fn
# #185218910 Setting up unit tests for React Vite

## What does this PR do?

This PR sets up Jest unit testing for React Vite frontend application.

## Description of Task to be completed?

Installing universal packages to be used while writing unit tests of Techsmith Ecommerce application.

## How should this be manually tested?

- Clone the repository using `git clone https://github.com/atlp-rwanda/e-comm-team-techsmith-fn`

- Checkout to the branch using `git checkout ch-tests-setup-185218910`

- Install node packages that the application depends on: `npm i`

- Create a `.env` file in the root directory and add all environment variables required to run the application. The required variables are listed in the `.env.example` file available in root directory of the application.

- Start the application development server: `npm start`

- Check out the components documentation: `npm run storybook`


## What are the relevant pivotal tracker/Trello stories?

- [#185218910](https://www.pivotaltracker.com/story/show/185218910)
